cmd/root.go

```diff
@@ -78,6 +78,8 @@ type Command struct {
 	telemetryPrefix            string
 	prometheus                 bool
 	prometheusNamespace        string
+	prometheusAddress          string
+	prometheusPort             string
```

Comment:
[minor] Since it’s prometheusPort, what about healthCheckPort instead of httpPort?